gap/SkeletalFinSetsForCAP.gi

```diff
@@ -932,6 +932,23 @@ AddClassifyingMorphismOfSubobjectWithGivenSubobjectClassifier( SkeletalFinSets,
       
 end );
 
+##
+AddMorphismsOfExternalHom( SkeletalFinSets,
+  function ( cat, A, B )
+    local hom_A_B, T;
+    
+    hom_A_B := ExponentialOnObjects( cat, A, B );
+    
+    T := TerminalObject( cat );
+    
+    return List( [ 0 .. Length( hom_A_B ) - 1 ],
```

Comment:
```suggestion
    return List( AsList( hom_A_B ),
```

Reply:
Currently this would lead to an obsolete Objectify, so I don't like this change.